samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -45,16 +47,25 @@ CONFIG_BT_CTLR_PRIVACY=n
 # Bluetooth mesh configuration
 CONFIG_BT_MESH=y
 CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=4
```

Comment:
Should there be  one more set for friendship? `BT_MESH_ADV_EXT_FRIEND_SEPARATE`

Reply:
I don't think so. Because the current defaults do not set FRIEND_SEPERATE. If user needs it they can set it as such.